From 93313e5be05589823c2d6056e4979a75375aab54 Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 2 Feb 2016 20:30:41 +0800
Subject: [PATCH 20/50] nx503a: ARM: dts: msm: Remove redundant IADC channel
 parameters

cherry-pick from 5cf5dad7f7e584bf98764ec9249cccb8e73a94f2

Remove redundant IADC channel parameters which are
not applicable on the IADC peripheral.

Signed-off-by: BambooIV <ningdisheng@gmail.com>
---
 arch/arm/boot/dts/NX503A/msm-pm8941.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/NX503A/msm-pm8941.dtsi b/arch/arm/boot/dts/NX503A/msm-pm8941.dtsi
index 1328620..3164c1f 100755
--- a/arch/arm/boot/dts/NX503A/msm-pm8941.dtsi
+++ b/arch/arm/boot/dts/NX503A/msm-pm8941.dtsi
@@ -898,10 +898,6 @@ CONFIG_ZTEMT_NX503A_CHARGE
 			label = "internal_rsense";
 			reg = <0>;
 			qcom,decimation = <0>;
-			qcom,pre-div-channel-scaling = <1>;
-			qcom,calibration-type = "absolute";
-			qcom,scale-function = <0>;
-			qcom,hw-settle-time = <0>;
 			qcom,fast-avg-setup = <0>;
 		};
 	};
-- 
1.9.1

